applications/PfemFluidDynamicsApplication/custom_strategies/strategies/nodal_two_step_v_p_strategy.h

```diff
@@ -2190,6 +2159,29 @@ namespace Kratos
 			KRATOS_CATCH("");
 		}
 
+	private:
+		void FillNodalSFDVector()
+		{
+
+			ModelPart &rModelPart = BaseType::GetModelPart();
+
+			//  #pragma omp parallel
+			//  	{
+			// 		ModelPart::NodeIterator NodesBegin;
+			// 		ModelPart::NodeIterator NodesEnd;
```

Comment:
remove?

Reply:
Lets leave it as it is just in case (it was like this before, we just moved it to a `private` section). @AFranci idea is to eventually do a clean up only PR.